NossoMercadoLivre/Services/UserService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NossoMercadoLivre.Models.Entities;
+using NossoMercadoLivre.Models.ViewModels;
+using NossoMercadoLivre.Repositories;
+using NossoMercadoLivre.Utils;
+using System;
+
+namespace NossoMercadoLivre.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _userRepository;
+
+        public UserService(IUserRepository userRepository)
+        {
+            _userRepository = userRepository;
+        }
+
+        public void Create(CreateUserViewModel user)
+        {
+            try
+            {
+                User newUser = new User
```

Comment:
*A sua api deve deixar claro o caminho que deve ser seguido pelo ponto do código que decide usá-la. Não espere que ninguém lembre de invocar nada. Faça de tudo para gerar obrigações. Quanto mais específico é seu código, menos democrático ele é.*

login e senha são obrigatórios :). Sugiro o uso do construtor. Se for obrigado a deixar um sem argumentos, deixa um comentário explicando que é exigência do framework.